### PR TITLE
adjustment in the ES6 JavaScript EventApi

### PR DESCRIPTION
**Pull request by serverKnights**

### CODE REVIEW

**Review by brotkrueml — COMMENTED** [review]
As we provide JS examples and not TS examples, the code examples should be corrected (and then the admonition adjusted accordingly). What do you think?

**Comment by serverKnights** [comment]
> As we provide JS examples and not TS examples, the code examples should be corrected (and then the admonition adjusted accordingly). What do you think?

Therefore the Context is JS in the examples the import should be containing the .js extention. i will create another request containing all the chainges spoken about in slack. https://typo3.slack.com/archives/C028JEPJL/p1705562996522919

**Comment by garvinhicking** [comment]
Great contribution, thanks for bringing this up. As noted in Slack I do think we should merge this.

File: Documentation/ApiOverview/Backend/JavaScript/EventApi/Index.rst [hunk 1]
```diff
@@ -10,6 +10,14 @@ The TYPO3 JavaScript Event API enables JavaScript developers to have a stable ev
 interface. The API takes care of common pitfalls like event delegation and clean
 event unbinding.
 
+.. warning::
+
+    When using :js:`import` statements, it is vital that you use the suffix
+    :file:`.js` to any import statements, when you are in the scope of `JavaScript/ES6`.
+    Only when you create your code in `TypeScript` (using :file:`.ts` suffix), you
+    need to omit the :file:`.js` filename in :js:`import` statements.
```
Comment (brotkrueml) [inline Documentation/ApiOverview/Backend/JavaScript/EventApi/Index.rst:18]:
```suggestion
    need to omit the :file:`.js` extension in :js:`import` statements.
```

**Review by garvinhicking — APPROVED** [review]

**Review by brotkrueml — APPROVED** [review]

**Comment by brotkrueml** [comment]
@serverKnights Thanks for the fix :+1:

**Pull request merged**